test/entrypoint.test.ts

```diff
@@ -1362,4 +1367,42 @@ describe('EntryPoint', function () {
       })
     })
   })
+
+  describe('ERC-165', function () {
+    it('should return true for EntryPoint interface IDs', async function () {
+      const epInterface = IEntryPoint__factory.createInterface()
```

Comment:
I miss a sample for: how a bundler can check the current entryPoint adddress it has is OK ?